site-packages/integralstor_utils/pki.py

```diff
@@ -102,6 +102,28 @@ def delete_ssl_certificate(name):
         return True, None
 
 
+def delete_all_ssl_certificates():
+    error_list = []
+    try:
+        ssl_certs, err = get_ssl_certificates()
+        if err:
+            raise Exception(err)
+
+        if ssl_certs:
+            for cert in ssl_certs:
+                for key in cert.keys():
```

Comment:
Well, I meant, should have been done with out the additional iteration of keys to get 'name`.